sonar-core/src/main/resources/org/sonar/core/qualityprofile/db/QualityProfileMapper.xml

```diff
@@ -151,6 +151,14 @@
     </where>
   </select>
 
+  <select id="countProjectsByProfile" resultType="org.sonar.core.qualityprofile.db.QualityProfileProjectCount">
```

Comment:
@jblievremont @julienlancelot do you think that this SQL request should filter on PROJECTS.ENABLED=true ?